slang-kotlin/src/main/java/com/sonarsource/slang/kotlin/KotlinSensor.java

```diff
@@ -63,18 +63,18 @@ public void execute(SensorContext sensorContext) {
 
   private void analyseFiles(SensorContext sensorContext, Iterable<InputFile> inputFiles, List<TreeVisitor<InputFileContext>> visitors) {
     for (InputFile inputFile : inputFiles) {
+      InputFileContext inputFileContext = new InputFileContext(sensorContext, inputFile);
       try {
-        analyseFile(sensorContext, inputFile, visitors);
+        analyseFile(inputFileContext, inputFile, visitors);
       } catch (IllegalStateException e) {
-        // FIXME Assert errors from kotlin parser
-      } catch (Throwable e) {
-        // FIXME Assert errors from kotlin parser
+        inputFileContext.reportError("Cannot parse file " + inputFile);
+      } catch (ParseException e) {
+        inputFileContext.reportError("Cannot parse file " + inputFile, e.getPosition());
```

Comment:
We should add test cases to `KotlinSensorTest` for that. We should probably also log a message.